__init__.py

```diff
@@ -183,17 +185,21 @@ def getDBPath():
     )
     if not db_path:
         raise IOError
-    db_path = unicode(db_path)
+    db_path = str(db_path)
     return db_path
 
 
 def getKindleVocabPath():
     try:
-        if platform == "win32":
+        if platform.system == "Windows":
```

Comment:
`platform` is actually a string, so this should be changed back to `if platform == "win32":` (https://docs.python.org/3.6/library/sys.html#sys.platform)

Reply:
ah - I see - stackoverflow led me astray :-), sorry. I'll work on this bit